orbit-2-viewmodel/src/test/java/com/babylon/orbit2/viewmodel/ViewModelExtensionsKtTest.kt

```diff
@@ -98,7 +101,7 @@ class ViewModelExtensionsKtTest {
         }
 
         // Used to trigger execution of onCreate
-        middleware.container.stateStream.observe { }
+        runBlocking { middleware.container.stateFlow.take(1).collect { } }
```

Comment:
We need the flow to complete so the code proceeds; this seemed like the easiest way to achieve that as we'll always get the initial state emitted so there's always one value and it arrives after onCreate is executed.